judge-d-server/src/main/java/dev/hltech/dredd/domain/JudgeD.java

```diff
@@ -36,9 +46,40 @@ public JudgeD(EnvironmentRepository environmentRepository, ServiceContractsRepos
         return getValidatorResult(validatedService, environmentContracts, validator);
     }
 
+    public <C, E> Map<ServiceVersion, EnvironmentValidatorResult> validatedServicesAgainstEnvironment(
+        List<ServiceContracts> validatedContracts,
+        String env,
```

Comment:
Here we forfeit possiblity of testing external environments. I think that batch should allow to include external environment (i.e. multiple environments) like in another endpoint (above)

Reply:
ok, will do it in another iteration.